### PR TITLE
DNS DSC

### PR DESCRIPTION
**Pull request by Xorima**
### Description

Adds ability to manage DNS using DSC, and to install a module using for
managing DSC

Signed-off-by: Jason Field <jason@avon-lea.co.uk>

### Issues Resolved

N/A

### Check List

- [ ] All tests pass. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/TESTING.MD>
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [ ] All commits have been signed for the Developer Certificate of Origin. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/CONTRIBUTING.MD>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -207,6 +207,66 @@ windows_dns "myservice.chef.test" do
 end
 ```
 
+
+## windows_dns_record
+
+Creates a DNS record for the given domain
+
+
```
Comment (tas50) [inline README.md]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -207,6 +207,66 @@ windows_dns "myservice.chef.test" do
...
+#### Actions
+
+- :create: creates/updates the DNS entry
+- :delete: deletes the DNS entry
+
+#### Properties
+
+- `name` : The name of the record to create, eg: calaserver01
```
Comment (tas50) [inline README.md]: From the resource this looks like record_name not name

File: README.md [hunk 3]
```diff
@@ -207,6 +207,66 @@ windows_dns "myservice.chef.test" do
...
+Creates an Active Directory Integrated DNS Zone on the local server
+
+#### Actions
+
+- :create: creates/updates the DNS Zone
+- :delete: deletes the DNS Zone
+
+#### Properties
+
+- `name` : The name of the zone to create, eg: calastone.com
```
Comment (tas50) [inline README.md]:
```suggestion
- `zone_name` : The name of the zone to create, eg: calastone.com
```

File: README.md [hunk 4]
```diff
@@ -207,6 +207,66 @@ windows_dns "myservice.chef.test" do
...
+  zone        'chef.local'
+  target      'arecord.chef.local'
+end
+```
+
+## dsc_module
+
+Installs a DSC module
+
+
```
Comment (tas50) [inline README.md]:
```suggestion
```

**Comment by Xorima** [comment]
What a pain, so doing a suggested fix commit doesn't add the DCO 👎

**Comment by Xorima** [comment]
Closing as we are going to use the windows_dns cookbook.

**Pull request closed without merging**